### PR TITLE
ENH: Enable HDU sorting in MultiDim

### LINKED ISSUE #556: Feature Request: Add option to have MultiDim sort extensions by name
This is a feature request by @jdavies-st . Currently, the drop-down menu sort extensions by number. I think it will be easy to add an user configurable item in `plugin_MultiDim.cfg` to enable this.

### LINKED ISSUE #557: Feature Request: MultiDim to auto start if multi-extension FITS is loaded
As requested by @jdavies-st -- If user loads a multi-extension FITS file, the loader can automatically starts `MultiDim` local plugin.

p.s. An easier alternative is just to have `MultiDim` set to always start when a channel is started.

### PR DESCRIPTION
**Pull request by pllim**
Fix #556 

Close #557 

cc @jdavies-st

### CODE REVIEW

File: ginga/rv/plugins/MultiDim.py [hunk 1]
```diff
@@ -91,7 +92,8 @@ def __init__(self, fv, fitsimage):
         # Load plugin preferences
         prefs = self.fv.get_preferences()
         self.settings = prefs.create_category('plugin_MultiDim')
-        self.settings.set_defaults(auto_start_naxis=False)
```
Comment (pllim) [inline ginga/rv/plugins/MultiDim.py]: @Cadair , just want to double check that removing this `auto_start_naxis` option does not break anything that SunPy is doing. It is not used here in Ginga. If you use it on your side, I would recommend to subclass this plugin instead and add the option over there.

Comment (pllim) [reply ginga/rv/plugins/MultiDim.py]: Speak now for forever hold your peace! :wink:

**Comment by ejeschke** [comment]
FWIW, I have been thinking that we should add an icon to the toolbar for opening `MultiDim`, since it is an oft-used plugin like `Preferences` or `FBrowser`...

**Comment by jdavies-st** [comment]
Tested this with some JWST files, and it looks good!

**Comment by pllim** [comment]
As per offline conversation with @jdavies-st , this is good to merge. Thanks, James and Eric!

**Pull request merged**